src/sensors/Sensor.py

```diff
@@ -42,24 +42,24 @@ def _processResponse(self) -> None:
 
     def _updateData(self) -> None:
         self._timestamp = time.monotonic()
-        self.dataChanged.emit()
+        self.data_changed.emit()
 
-    dataChanged = Signal()
+    data_changed = Signal()
```

Comment:
I am a bit torn here. If the Signal `dataChanged` is changed to `data_changed`, then the Property `unitMap` should also be `unit_map`. But for me, both of them map to things properties/"things" in QML, and QML expects its own code style. For example the eventhandler for a signal with the name `data_changed` would become `onData_changed` which looks weird (whereas the eventhandler for `dataChanged` becomes `onDataChanged`).

Reply:
That's entirely reasonable, if it lines up with the QML then I will change those back and disable linting for them instead.